### PR TITLE
Faster build

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
Turns out the build was pulling from node_modules, which is why it was so slow. There are a few other updates I'd like to make, but this should do for now.

This UX pull request is referencing this branch:
https://github.com/blackbaud/skyux2/pull/1294

### CODE REVIEW

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-BobbyEarl Alright, this is finally ready for review.

File: config/webpack/common.webpack.config.js [hunk 1]
```diff
@@ -75,8 +75,9 @@ function getWebpackConfig(skyPagesConfig, argv = {}) {
       rules: [
         {
           enforce: 'pre',
-          test: /runtime\/config\.ts$/,
-          loader: outPath('loader', 'sky-app-config')
+          test: /config\.ts$/,
+          loader: outPath('loader', 'sky-app-config'),
+          include: outPath('runtime')
```
Comment (Blackbaud-BobbyEarl) [inline config/webpack/common.webpack.config.js]: Just curious about the different syntax here.

Comment (Blackbaud-SteveBrush) [reply config/webpack/common.webpack.config.js]: it will ONLY look in the root's runtime folder, instead of "any" runtime folder.

Comment (Blackbaud-SteveBrush) [reply config/webpack/common.webpack.config.js]: Not a huge time saver, but recommended by Webpack docs to be more specific with loaders. https://webpack.js.org/guides/build-performance/#loaders

**Pull request merged**